src/ai/backend/client/func/vfolder.py

```diff
@@ -297,6 +298,13 @@ async def delete_trash(self) -> Mapping[str, Any]:
         async with rqst.fetch():
             return {}
 
+    @api_function
+    async def force_delete(self) -> dict[str, Any]:
+        await self.update_id_by_name()
+        rqst = Request("DELETE", "/folders/{0}/force".format(self.request_key))
+        async with rqst.fetch() as resp:
+            return {"success": resp.status == HTTPStatus.NO_CONTENT}
```

Comment:
TODO: align the output of SDK functions

Reply:
But I think the caller cannot decide whether the request has succeeded through an empty dictionary (`{}`).